docs/tutorial/part03.rst

```diff
@@ -2,5 +2,245 @@
+          "address": "MEI SERVICES LIMITED ROOM E; 6TH FLOOR; EASTERN COMMERCIAL CENTRE; 395-399 HENNESSY ROAD HONG KONG",
+          "jurisdiction": "SAM",
+          "service_provider": "Mossack Fonseca",
+          "countries": "Hong Kong",
+          "jurisdiction_description": "Samoa",
+          "valid_until": "The Panama Papers data is current through 2015",
+          "ibcRUC": "25475",
+          "name": "JIE LUN INVESTMENT LIMITED",
+          "country_codes": "HKG",
+          "incorporation_date": "10-APR-2006",
+          "node_id": "10000020",
+          "status": "Active,
+        }
+
+Therefore, the structure of the ``Entity`` class should be::
+
+    class Entity(StructuredNode):
+        sourceID                           = StringProperty()
+        address                            = StringProperty()
+        jurisdiction                       = StringProperty()
+        service_provider                   = StringProperty()
+        countries                          = StringProperty()
+        jurisdiction_description           = StringProperty()
+        valid_until                        = StringProperty()
+        ibcRUC                             = StringProperty()
+        name                               = StringProperty()
+        country_codes                      = StringProperty()
+        incorporation_date                 = StringProperty()
+        node_id                            = StringProperty(index = True)
+        status                             = StringProperty()
+
+4. Add the relationships for the node::
+
+    officers                 = RelationshipFrom('.officer.Officer', 'OFFICER_OF')
+    intermediaries           = RelationshipFrom('.intermediary.Intermediary', 'INTERMEDIARY_OF')
+    addresses                = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+    others                   = RelationshipFrom('.other.Other', 'CONNECTED_TO')
+
+.. [*]  You can find the nodes relation by calling on the database: CALL db.schema(). To access your local database click HERE_. note: you need to have it up and running to work. 
+
+    .. _HERE: http://localhost:7474/browser/ 
+
+
+
+How **RelationshipFrom** and **RelationshipTo** works:
+
+        1. The first parameter is the type of node you want to connect. e.g ``.officer.Officer``
+        2. The second parameter is the relationship type. e.g. ``OFFICER_OF``
+
+
+* **RelationshipFrom** is an INCOMING relationship 
+* **RelationshipTo**  is an OUTGOING relationship 
+* Also, there is one call **Relationship** which can be either
+
+If  **RelationshipFrom**  be illustrated,  the output would be something like: 
+
+.. image:: images/relfrom.png
+   :width: 100%
+   :alt: alternate text
+
+
+Repeat these steps for each node class you wish to create. On this program those would be: ``address.py`` , ``intermediary.py`` , ``officer.py``, and ``other.py``. You must add the following code to each of the files:

```

Comment:
After `program` I think we need a comma but I'm not sure; what do you think? 🤔 
After the entities are an extra space before the comma

Reply:
You are right!